istioctl/cmd/add-to-mesh.go

```diff
@@ -172,8 +173,9 @@ func svcMeshifyCmd() *cobra.Command {
 	var opts clioptions.ControlPlaneOptions
 
 	cmd := &cobra.Command{
-		Use:   "service <service>",
-		Short: "Add Service to Istio service mesh",
+		Use:     "service <svc>",
```

Comment:
Again, I think the `use` line should stay the same, this just says the argument must be a Kubernetes _Service_.